docs/dev-guide.md

```diff
@@ -66,6 +66,8 @@ make docker_http
 
 > N.B. Mainflux creates `FROM scratch` docker containers which as compact and small in size.
 
+> N.B. Rebuilding of all Docker containers or the `things-db` and `users-db` containers separately will cause data loss. All your users, things, channels and connections between them will be lost! If you need to retain the data between the container rebuilds you can attach volume to the `things-db` and `users-db` containers. Check the official docs on how to use volumes [here](https://docs.docker.com/storage/volumes/) and [here](https://docs.docker.com/compose/compose-file/#volumes).
```

Comment:
I know that the `things-db` and `users-db` are vanilla PostgreSQL images. I just added this comment so that it's clear that if someone rebuilds the containers he or she will lose his/hers data and added links to the needed changes to persist the data.
I think it is good to have something like this in the guide so that we are on the safe side if someone complains about data loss.
Of course this is a dev setup and we don't care about reliability and data persistence.

Reply:
@chombium yes, I agree that it is useful to have a warning, just that we have to make it clear to the people that these DB images are not built by us, and thus not rebuildable (as doc currently suggest).

Reply:
@drasko I've refined the text a bit so that, now it is explicitly written that the containers are built from vanilla PostgreSQL image, downloaded from Docker Hub and t are not built by us, and thus not rebuildable (ashat we use this setup for development and we can not guarantee data persistence.